tests/test_ajd.py

```diff
@@ -24,9 +39,15 @@ def test_rjd():
 
 def test_pham():
     """Test pham's ajd"""
-    covmats, diags, A = generate_cov(100, 3)
+    Nt = 100
+    covmats, diags, A = generate_cov(Nt, 3)
     V, D = ajd_pham(covmats)
 
+    w = 5 * np.ones(Nt)
+    Vw, Dw = ajd_pham(covmats, sample_weight=w)
+    assert_array_equal(V, Vw)
+    assert_array_equal(D, Dw)
```

Comment:
what I would test is that if the weight of one covmat is 0 it's like it was not passed. It's what sklearn does for testing.